scripts/wait_for_deletion.sh

```diff
@@ -44,10 +44,13 @@ poll_interval=4
 while true; do
   # Everything under the namespace needs to be removed after app/uninstall
   echo "INFO Checking if $kind were deleted"
-  resources=$(kubectl get $kind \
-    --namespace="$namespace" \
-    -o=json \
-    | jq -r '.items[] | "\(.kind)/\(.metadata.name)"')
+  resources="$(
+    (kubectl get $kind \
+     --namespace="$namespace" \
+     -o=json \
+     | jq -r '.items[] | "\(.kind)/\(.metadata.name)"' \
+    ) \
+    || echo "Unexpected error. Will retry")"
 
   res_count=$(echo $resources | wc -w)
```

Comment:
Nit `echo "Unexpected error. Will retry" | wc -w` will make `"INFO Remaining: $res_count"` be `INFO Remaining: 4`

Reply:
The parentheses make this statement slightly easier to read so I'm gonna keep them.

Reply:
Good call. Made changes to handle this error message specifically, for a friendly log output.